slapp/templates/pages/map.html

```diff
@@ -31,10 +35,19 @@ <h4 class="text-center">Karte</h4>
         <p>
           Außerdem können Sie eine <strong>kostenminimierende Energiesystemoptimierung</strong> für eine oder mehrere Gemeinden durchrechnen lassen.
         </p>
-      </div>
-      <div class="col-9">
-        <div id="legend" class="legend"></div>
-        <div id="map"></div>
+        <div class="wizard__next">
+          <button hx-post="{% url 'explorer:details' %}"
```

Comment:
Here wizard href is hardcoded, which makes it easier IMO, instead of calling next_step url and dynamically load next view